extensions/wikia/Email/EmailController.class.php

```diff
@@ -507,6 +504,10 @@ private function traverseThrough( $mobileApplications ) {
 					break;
 				}
 			}
+
+			if ( !empty($mobileApplicationsLinks) ) {
```

Comment:
Spaces around `empty` argument!  Tired of hearing that yet? ;)  If so check out the tips in the HowTo article I just created:

https://wikia-inc.atlassian.net/wiki/display/SOC/How+to+style+PHP+code